llama_parse/base.py

```diff
@@ -22,7 +24,7 @@ class LlamaParse(BasePydanticReader):
 
     api_key: str = Field(default="", description="The API key for the LlamaParse API.")
     base_url: str = Field(
-        default="https://api.cloud.llamaindex.ai/api/parsing",
+        default=DEFAULT_BASE_URL,
```

Comment:
I think you need to append `api/parsing` in the httpx call now?

Reply:
oh whoops lol